src/adapters/indexExchange.js

```diff
@@ -361,7 +361,7 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     }
 
     //bidmanager.setExpectedBidsCount(ADAPTER_CODE, expectedBids);
-    adloader.loadScript(cygnus_index_start());
+    adloader.loadScript(cygnus_index_start(), null, true);
```

Comment:
This should not be cached, the ad payload is returned in the script, I'll update.